docs/root/configuration/listeners/network_filters/rbac_filter.rst

```diff
@@ -23,6 +23,9 @@ Statistics
 
 The RBAC network filter outputs statistics in the *<stat_prefix>.rbac.* namespace.
 
+For the shadow rule statistics `shadow_allowed` and `shadow_denied`, the :ref:`shadow_rules_stat_prefix <envoy_v3_api_field_extensions.filters.http.rbac.v3.RBAC.shadow_rules_stat_prefix>`
```

Comment:
Copy/paste issue here for the field? Should point to network filter? Same below?

Reply:
Ah, thanks for catching. Updated both.